tests/integration/test_shape.py

```diff
@@ -44,18 +45,22 @@ def test():
     d = y.shape[1] / 3
     assert d == 2.0
 
-    # TODO: Reshape is not yet supported in Legate
-    # x = num.random.random((2,3,4))
-    # y = x.reshape((4,3,2))
-    # assert(y.shape == (4,3,2))
-    # assert(y.size == x.size)
-    # pos = 0
-    # for a in range(0,y.size):
-    #    print(pos, y.flat[pos], x.flat[pos])
-    #    assert(y.flat[pos] == x.flat[pos])
-    #    pos = pos+1
-    return
+
+# TODO: Reshape is not yet supported in Legate
```

Comment:
This is no longer true, so this test could be put back. maybe you should try to remove the decorator and see if passes.

Reply:
restored in https://github.com/nv-legate/cunumeric/pull/297/commits/4f3d9574bc46b4d09eb4fca8b1cbc23ab595e4bf